contracts/testing_utils/DummyPillarToken.sol

```diff
@@ -0,0 +1,10 @@
+// SPDX-License-Identifier: MIT
```

Comment:
You wouldn't need this to be part of the repo.

Reply:
This is required for testing.

Reply:
Required for testing